src/components/App.tsx

```diff
@@ -180,9 +180,13 @@ export class App extends React.Component<any, AppState> {
       return;
     }
     copyToAppClipboard(elements, this.state);
-    elements = deleteSelectedElements(elements, this.state);
+    const { elements: nextElements, appState } = deleteSelectedElements(
+      elements,
+      this.state,
+    );
+    elements = nextElements;
     history.resumeRecording();
-    this.setState({});
+    this.setState({ ...appState });
```

Comment:
should this be before or after `history.resumeRecording()`?

Reply:
Doesn't matter in practice, history is checked during componentDidUpdate, so both work. We put it before so it's more clear.